src/Worker/Core/Shims/TaskOrchestrationContextWrapper.cs

```diff
@@ -84,7 +85,7 @@ public TaskOrchestrationContextWrapper(
     /// <summary>
     /// Gets the configuration settings for the orchestration.
     /// </summary>
-    public override IDictionary<string, object?> Properties => this.properties;
+    public override IDictionary<string, object?> Properties { get; } = new Dictionary<string, object?>();
```

Comment:
You are always setting this in the ctor. No need to inline an initialization.

```suggestion
    public override IDictionary<string, object?> Properties { get; };
```